deploy/kubernetes/base/server-service.yaml

```
@@ -11,14 +11,14 @@ metadata:
   name: server
 spec:
   ports:
-  - name: "443"
+  - name: "api"
```

nit: If you'd like to be compatible with running on an Istio-enabled cluster, it'd be best to use [their naming conventions for ports](https://istio.io/docs/ops/configuration/traffic-management/protocol-selection/).

```suggestion
  - name: "https-api"
```

---

Thanks